specs/v1.yaml

```diff
@@ -364,6 +377,89 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /v1/grants/unintegrated:
```

Comment:
i personally would like unintegrated grants to live as a separate resource in our APIs: e.g. `unintegrated-grants` so this becomes: `v1/unintegrated-grants`